tox.ini

```diff
@@ -31,10 +31,21 @@ deps = autopep8
 commands = autopep8 --ignore E309,E501 -a -i -r luigi test examples bin
 
 [testenv:docs]
-# to call this use `tox -e docs -- html`
+# Build documentation using sphinx.
+# Call this using `tox -e docs -- html`.
 deps =
   sqlalchemy
   Sphinx
   sphinx_rtd_theme
 changedir = doc
-commands = /usr/bin/make {posargs}
+commands = /usr/bin/make SPHINXOPTS='-W' {posargs}
```

Comment:
I saw the PR.
either way works.
I'll wait for the merge for #697 and then I'll rebase.